entrypoint.sh

```diff
@@ -18,8 +18,12 @@ docker_network_rule=$([ ! -z "$docker_network" ] && echo "! -d $docker_network"
 iptables -I OUTPUT ! -o $interface -m mark ! --mark $(wg show $interface fwmark) -m addrtype ! --dst-type LOCAL $docker_network_rule -j REJECT
 
 docker6_network="$(ip -o addr show dev eth0 | awk '$3 == "inet6" {print $4}')"
-docker6_network_rule=$([ ! -z "$docker6_network" ] && echo "! -d $docker6_network" || echo "")
-ip6tables -I OUTPUT ! -o $interface -m mark ! --mark $(wg show $interface fwmark) -m addrtype ! --dst-type LOCAL $docker6_network_rule -j REJECT
+if [[ -z "${docker6_network}" ]]; then
```

Comment:
Hrmm, judging from line 6, should this line just be:
```
if [[ -z $docker6_network ]]; then
```

Reply:
hmm, is there any difference other than the quoting? I just quoted as out of habit to prevent shell splitting/expansion.

Reply:
I'm cool either way, as long as the two lines (6 and 21) are consistent 👍

Reply:
Sounds good, updated to keep them consistent.